playbooks/pre-install-playbook/roles/viya-ark.preinstall/tasks/pre.package_version_check.yml

```diff
@@ -1,30 +1,30 @@
 #
-# Copyright (c) 2019-2023, SAS Institute Inc., Cary, NC, USA.  All Rights Reserved.
+# Copyright (c) 2019-2025, SAS Institute Inc., Cary, NC, USA.  All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0
 #
 ---
 ####################################################################
-## Required Package Version for RHEL
+## Required Package Version for RHEL or Oracle Linux
 ####################################################################
-# This task block should only run on RHEL systems, as it uses yum
+# This task block should only run on RHEL/Oracle systems, as it uses yum
 # to query available package versions.
 
 - block:
-  ## RHEL packages with a minimum version requirement
+  ## RHEL/Oracle packages with a minimum version requirement
 
   - name: Default valid installed package flag to false
     set_fact:
       package_meets_requirement: False
      
   # Is the package already installed at the required level?
-  - name: Check for installed RHEL package '{{ item.name }}'
+  - name: Check for installed RHEL/Oracle package '{{ item.name }}'
```

Comment:
...and so on to revert back to previous wording

Reply:
The primary reason for introducing explicit references to oracle was so that the output logs wouldn't present any confusion when "RHEL" task name output is seen.   The comments in the file will now clarify RHEL and RHEL compatible systems, but that's not visible in the playbook output.   I am taking any OS distribution reference out of the task names and trust users know the context of their OS env and can live without it being explicit in these task names.

Reply:
good call, removing RHEL from the ansible task name that is seen in the log is probably best